invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/Identifiers/PIDField.js

```diff
@@ -407,40 +442,72 @@ class CustomPIDField extends Component {
     }
 
     const hasManagedIdentifier = managedIdentifier !== "";
+    const hasUnmanagedIdentifier = unmanagedIdentifier !== "";
```

Comment:
why do we set empty string ?

Reply:
The initialization happens above and here we check if the value is not empty string. I am confused about the question :)